### ISSUE TITLE
Player models sporadically disappear

### ISSUE THREAD

**Issue by no-lex** [project-imprimis/libprimis#177]
Sometime between 
```
commit 93ea28e1a18461ff8b8a14f6831c47710f79914e (HEAD -> master)
Author: Alex Foster <123afoster@gmail.com>
Date:   Thu Nov 5 20:12:54 2020 -0800

    apply debugfullscreen for aa debug, closes #156
    
    adds debugfullscreen to engine.h as a result
```
and
```
commit 432c504d6add047f00ee9b0679cc301694ee01f0 (HEAD -> master)
Author: Alex Foster <123afoster@gmail.com>
Date:   Thu Nov 12 21:32:19 2020 -0800

    adopt std::vector in cfpolys struct for #45
    
    in octa.cpp
```
something in the model handling code broke, and models randomly (there is a pattern but it's not clear) fail to render. They are still present, because their above head billboards still show and they still can be killed.

**Comment by no-lex** [comment]
Appears to be caused by 1f6792d21e730587d76ed7fc42370d8bda93bb4e

**Comment by no-lex** [comment]
closed by 05fbfe5
